openlibrary/templates/books/custom_carousel.html

```diff
@@ -1,16 +1,19 @@
 $def with(books=[], title="", url="", key="", min_books=1, load_more=None, test=False)
 
 $def render_carousel_cover(book, lazy):
+  $ fallback_cover = 'https://openlibrary.org/images/icons/avatar_book.png'
+  $ cover_host = '//covers.openlibrary.org'
   $ url = book.get('key') or book.url
   $ title = book.title
+
   $if book.get('cover_url'):
     $ cover_url = book.get('cover_url')
   $elif book.get('cover_id'):
     $ cover_url = '%s/b/id/%s-M.jpg'%(cover_host, book.get('cover_id'))
   $elif book.get('ia'):
-    $ cover_url = '%s/b/ia/%s-M.jpg?default=%s'%(cover_host, book.get('ia')[0], fallback_cover)
+    $ cover_url = '%s/b/ia/%s-M.jpg?default=%s'%(cover_host, book.get('ia')[0], False)
```

Comment:
```suggestion
    $ cover_url = '%s/b/ia/%s-M.jpg' % (cover_host, book.get('ia')[0])
```

Reply:
The fallback cover if set will prevent the cover with title and author from showing.

Reply:
Ah, ok; then can we remove `?default` entirely?

Reply:
```suggestion
    $ cover_url = '%s/b/ia/%s-M.jpg' % (cover_host, book.get('ia')[0])
```

Reply:
@jdlrobson could I ask you to accept this suggestion? (I don't have permission)